ioctl/cmd/action/action.go

```diff
@@ -132,6 +156,15 @@ func execute(contract string, amount *big.Int, bytecode []byte) (err error) {
 		log.L().Error("error when invoke an execution", zap.Error(err))
 		return
 	}
+	if gasLimit == 300000 {
```

Comment:
why 300000?

Reply:
Is there a better way to check if user sets gasLimitFlag?need to find out

Reply:
Default to 0, as long as 0, we will estimate?

Reply:
ok